.github/workflows/build.yml

```diff
@@ -50,10 +50,77 @@ jobs:
       - name: "Install Python dependencies for PDF Generator"
         run: pip install -r service-python/pdfgenerator/src/requirements.txt
 
+      - name: Restore cache
+        uses: actions/cache@v3
+        with:
+          path: |
+            ~/.gradle
+          restore-keys: |
+            v1-dependencies-${{ hashFiles('build.gradle') }}
+            v1-dependencies-
+          key: v1-dependencies-${{ hashFiles('build.gradle') }}
+
+#      - name: Setup gradle
+# The `dependencies` gradle task tries to pull VA's cookiecutter jar,
+# which fails and is not needed
+#        run: ./gradlew dependencies
+
```

Comment:
Commented this out b/c it tries to pull VA's cookiecutter jar, which fails and is not needed